src/main/java/com/arpnetworking/metrics/mad/Main.java

```diff
@@ -250,14 +250,15 @@ private Injector launchGuice(final ActorSystem actorSystem) {
         }
 
         // Instantiate the metrics factory
+        final String sinkHost = "0.0.0.0".equals(_configuration.getHttpHost()) ? "localhost" : _configuration.getHttpHost();
```

Comment:
Should we map 0.0.0.0 to 127.0.0.1 instead of the name localhost? Not sure it matters.